src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/item-list-preview.component.html

```diff
@@ -14,7 +14,7 @@ <h2 [innerHTML]="dsoTitle" [ngClass]="{'lead': true,'text-muted': !item.firstMet
           <span class="text-muted">
             <ds-truncatable-part [id]="item.id" [minLines]="1">
               (<span *ngIf="item.hasMetadata('dc.publisher')" class="item-list-publisher"
-                [innerHTML]="item.firstMetadataValue('dc.publisher') + ', '"></span>
+                [innerHTML]="item.firstMetadataValue('dc.publisher', undefined, true) + ', '"></span>
               <span class="item-list-date"
                 [innerHTML]="item.firstMetadataValue('dc.date.issued') || ('mydspace.results.no-date' | translate)"></span>)
```

Comment:
I think this line needs to be changed to `item.firstMetadataValue('dc.date.issued', undefined, true)` to align with #4737  Or am I missing something?